definitions/infra-processes/definition.yml

```diff
@@ -0,0 +1,32 @@
+domain: INFRA
+type: PROCESS
+
+synthesis:
+  name: commandName
+  identifier: commandLine
+  encodeIdentifierInGUID: true
+  conditions:
+  - attribute: eventType
+    value: ProcessSample
+
+  tags:
+    hostname:
+      multiValue: true
+    app_support_team:
+      multiValue: true
+    datacenter:
+      multiValue: true
+    environment:
+      multiValue: true
+    userName:
+      multiValue: true
+    processDisplayName:
+      multiValue: true
+
+dashboardTemplates:
+  newRelic:
+    template: dashboard.json
+
+configuration:
+  entityExpirationTime: DAILY
```

Comment:
Have you considered shorter TTLs? (e.g. I assume in the general case a process that is restarting does so in shorter timespans, so it may be enough to keep the entity alive for 4h)